tests/shipping/models/Delivery - Paraguay Geolocation.model.js

```diff
@@ -0,0 +1,53 @@
+import { visitAndClearCookies, setup } from '../../../utils'
+import {
+  getRandomEmail,
+  fillEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { SKUS } from '../../../utils/constants'
+import { selectCountry, goToPayment } from '../../../utils/shipping-actions'
+import { payWithBoleto, completePurchase } from '../../../utils/payment-actions'
+
+export default function test(account) {
+  describe(`Delivery - Paraguay - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.PARAGUAY_DELIVERY], salesChannel: 2, account })
+
+      fillEmail(email)
+      fillProfile()
+
+      selectCountry('PRY')
+
+      cy.get('#ship-addressQuery').type('Avenida Brasilia')
```

Comment:
I think that we should mock this return and explicitly state that the postal code is null, just in case that GMaps fixes this address

Wdyt?

Reply:
looks good to me!

Reply:
done in 723c136 :-)